### PR TITLE
Thai/user model (branched from leian/user)

### PR DESCRIPTION
**Pull request by leian7**
fixes issue #17 

## Changes in this PR.

- [X] Tests for the custom user model
- [X] Custom user model implemented 
- [x] Add comments (primarily to tests)

## Testing this PR.

1. Start up an instance of the StreamWebs docker container
2. Log into the container w/ docker-compose run web bash
3. cd into /streamwebs_frontend
4. Run python manage.py test streamwebs/tests/models

### Expected Output.

All tests should pass.

@athai @Kennric @LyonesGamer @matthewrsj @ElijahCaine

### CODE REVIEW

**Comment by ElijahCaine** [comment]
The CI is failing, there is a merge conflict, and the checklist is incomplete so I'm removing the 'ready for review' tag from this PR.

**Comment by athai** [comment]
Looks good to me! +1

File: streamwebs_frontend/streamwebs_frontend/settings.py [hunk 1]
```diff
@@ -129,6 +129,12 @@
 
 STATIC_URL = '/static/'
 
+# Temporary list of schools
+SCHOOL_CHOICES = (
```
Comment (Kennric) [inline streamwebs_frontend/streamwebs_frontend/settings.py]: Leave this empty - if you need a list for a test, set the value in the test itself.

File: streamwebs_frontend/streamwebs/templates/streamwebs/index.html [hunk 2]
```diff
@@ -1,12 +0,0 @@
-<!DOCTYPE html>
-<html>
-    <head>
-        <title>StreamWebs</title>
-    </head>
-
-    <body>
-        <h1>Welcome to StreamWebs</h1>
-    </body>
-</html>
```
Comment (Kennric) [inline streamwebs_frontend/streamwebs/templates/streamwebs/index.html]: Is this template replaced somewhere else?

File: docs/source/usage.rst [hunk 3]
```diff
@@ -1,114 +0,0 @@
-.. _usage:
-
-================
```
Comment (Kennric) [inline docs/source/usage.rst]: Are these docs replaced somewhere else?

Comment (leian7) [inline docs/source/usage.rst]: @Kennric Yes, the good version of these docs are already merged into develop.

File: streamwebs_frontend/streamwebs/templates/streamwebs/index.html (hunk 2 above)
Comment (leian7) [inline streamwebs_frontend/streamwebs/templates/streamwebs/index.html]: Yes, it's already been merged into develop from my index branch.

File: streamwebs_frontend/streamwebs_frontend/settings.py [hunk 4]
```diff
@@ -129,12 +129,8 @@
 
 STATIC_URL = '/static/'
 
-# Temporary list of schools
-SCHOOL_CHOICES = (
-    ('a', 'School A'),
-    ('b', 'School B'),
-    ('c', 'School C'),
-)
+# List of schools
```
Comment (Kennric) [inline streamwebs_frontend/streamwebs_frontend/settings.py]: One last note on this, sorry but I should have mentioned it before - empty values in the setting file should document their format, so people will know what to fill them with.

**Comment by Kennric** [comment]
Other than above note on documenting the schools list, this is good. Fix that and I'll +1

**Pull request merged**